guides/v2.1/ui_comp_guide/concepts/ui_comp_linking_concept.md

```diff
@@ -26,10 +26,11 @@ Linking properties are set in [UI components configuration files]({{page.baseurl
 
```

Comment:
Please, remove everywhere phrase about observable or add information about method `set` and `track` property.

Reply:
The fact that the imports, exports, links and listens only are triggered for changed values on observables is so important!
After reading the original documentation it was extremely frustrating that my code examples didn't work. With the new information I think they will be much more helpful to developers who want to learn about this.

Reply:
Oh, I don't want to add more information about how to configure `default.tracks` or `track()` or `ko.observable` here since that is another topic, I don't have the time to do that. All I want to do with this PR is fix the obvious omissions.